multimetric/__main__.py

```diff
@@ -76,44 +73,17 @@ def ArgParser():
         type=int,
         default=1,
         help="Run x jobs in parallel")
+    parser.add_argument(
+        "--ignore_lexer_errors",
+        default=False,
```

Comment:
I would actually make this true by default and issue a warning when lexing fails

Reply:
Haha, that's what I'd originally thought too. I was afraid it was too presumptuous of me. Will update PR.